components/serializers.py

```diff
@@ -27,6 +26,8 @@ class Meta:
             "description",
             "type",
             "supported_catalog_versions",
+            "catalog",
```

Comment:
Need to update with dev. This will break since the component model in dev no longer has a `catalog` field